bittensor/core/extrinsics/transfer.py

```diff
@@ -15,19 +15,81 @@
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
-from typing import Union, TYPE_CHECKING
+from typing import Dict, Tuple, Optional, Union, TYPE_CHECKING
```

Comment:
You don't need `Dict` or `Tuple`.

Reply:
I need this because of the linter doesn't want to see dict and tuple instead of Dict and Tuple as type annotation. It's complaints.